dart/lib/src/sentry_options.dart

```diff
@@ -256,6 +274,8 @@ class SentryOptions {
     sdk = SdkVersion(name: sdkName(platformChecker.isWeb), version: sdkVersion);
     sdk.addPackage('pub:sentry', sdkVersion);
   }
+  
+  SentryOptions.empty();
```

Comment:
Since this seems to be only for internal use?
```suggestion
  @internal
  SentryOptions.empty();
```

Reply:
Ok, now we have an issue in tests and analyze, will try to resolve.

```
info • The member 'SentryOptions.empty' can only be used within its package • dio/test/mocks/mock_hub.dart:19:20  invalid_use_of_internal_member
```